THIRD-PARTY-NOTICES.txt

```diff
@@ -0,0 +1,32 @@
+This software depends on external packages and source code.

```

Comment:
Use this: 

```
This product includes copyrighted software from third parties that may be
distributed under different licenses than this product as given below.

In case we missed to list a specific software and/or underlying license,
we appreciate your upfront notice.

This product may include copyrighted software that is distributed under a
license that requires us to provide the source code of that software and
instructions on how to build and install it, such as the GNU GPL or GNU
LGPL or other reciprocal licenses. Unless otherwise stated the original
source code of such software has not been modified. As recipient and user of
this product you may obtain the complete source code for such copyrighted
software as well as build and install instructions from us for a period of
three years starting with the reception of the product and at no charge.

In any of such cases, please send an email to opensource@zeiss.com referencing
your product's name, version, and manufacturing ZEISS entity.

===============================================================================

This document uses standard identifiers for software packages and licenses as
specified by the SPDX and Package URL groups:

The licenses identifiers are SPDX license expressions using
https://spdx.dev/licenses/. The full SPDX specification is at
https://spdx.dev/specifications/
```

Reply:
Done